microsoft/testsuites/lis/lissuite.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Logger, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import CentOs, Redhat
+from lisa.sut_orchestrator.azure.tools import LisDriver
+from lisa.testsuite import simple_requirement
+from lisa.tools import Wget
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="lis",
+    category="functional",
+    description="""
+    ##################################
+    """,
+    requirement=simple_requirement(
+        supported_os=[
+            CentOs,
+            Redhat,
+        ]
+    ),
+)
+class Lis(TestSuite):  # noqa
+    @TestCaseMetadata(
+        description="""
+            #################
+        """,
+        priority=1,
+    )
+    def verify_lis_driver_version(self, node: Node, log: Logger) -> None:
+        wget_tool = node.tools[Wget]
+        node.tools[LisDriver]
+
+        # Checking for RMP
+        rpm_status = node.execute(
+            'rpm -qa | grep "kmod-microsoft-hyper-v" && rpm -qa '
+            '| grep "microsoft-hyper-v"'
+        ).exit_code
+        if rpm_status != 0:
+            raise SkippedException("No LIS RPM's are detected. Skipping test.")
+
+        # Capturing LIS version from the node
+        version = node.execute(
+            """modinfo "hv_vmbus" | grep version: | head -1 | awk '{print $2}'""",
```

Comment:
use modinfo tool to get version

Reply:
I'm having trouble getting the modinfo tool to print out the lis version. This is the command i'm using: `
print("get ver: ", modinfo_tool.get_version("hv_vmbus"))`
and this is the output i'm getting: 
`2021-11-30 16:16:17.096[14604][INFO] lisa.stdout get ver:  `

Reply:
What's the raw output? You can check the code, and modify the regexp to include the new format. Don't forget to add the new format in comment examples.

Reply:
The raw output usually doesn't print out the LIS version. I've only seen one run where it actually printed out the LIS version. This is a snippet from a recent run: 
`2021-11-30 20:20:02.601[9792][DEBUG] lisa.env[generated_0].node[0].tool[modinfo] loaded cached result for command: [modinfo hv_vmbus]
2021-11-30 20:20:02.601[9792][INFO] lisa.stdout info:  filename:       /lib/modules/3.10.0-862.11.6.el7.x86_64/kernel/drivers/hv/hv_vmbus.ko.xz
2021-11-30 20:20:02.605[9792][INFO] lisa.stdout license:        GPL
2021-11-30 20:20:02.606[9792][INFO] lisa.stdout retpoline:      Y
2021-11-30 20:20:02.606[9792][INFO] lisa.stdout rhelversion:    7.5
2021-11-30 20:20:02.607[9792][INFO] lisa.stdout srcversion:     FDEC375DD4661BC80D39F62
2021-11-30 20:20:02.608[9792][INFO] lisa.stdout alias:          acpi*:VMBus:*
2021-11-30 20:20:02.609[9792][INFO] lisa.stdout alias:          acpi*:VMBUS:*
2021-11-30 20:20:02.610[9792][INFO] lisa.stdout depends:        
2021-11-30 20:20:02.610[9792][INFO] lisa.stdout intree:         Y
2021-11-30 20:20:02.611[9792][INFO] lisa.stdout vermagic:       3.10.0-862.11.6.el7.x86_64 SMP mod_unload modversions 
2021-11-30 20:20:02.612[9792][INFO] lisa.stdout signer:         CentOS Linux kernel signing key
2021-11-30 20:20:02.613[9792][INFO] lisa.stdout sig_key:        D4:11:5F:11:00:55:DB:56:C8:D6:05:AB:75:21:73:CF:B1:AC:54:D8
2021-11-30 20:20:02.614[9792][INFO] lisa.stdout sig_hashalgo:   sha256
2021-11-30 20:20:02.615[9792][INFO] lisa.stdout get ver:  7.5`

Reply:
The result is cached, and only printed in first run. It needs to look up in the node's log to find the previous runs of this command.

Reply:
The code now uses modinfo properly.